Automattic/studio
# Studio: Add translations for 1.2.0 Studio release

pfHvTO-fJ-p2

## Proposed Changes

This PR updates translations for the 1.2.0 Studio release.

## Testing Instructions



* Start the app with `nvm use && npm install && npm start`
* Confirm that the app works in different languages (use languages switcher to switch languages)

## Pre-merge Checklist
